velox/expression/tests/CastExprTest.cpp

```diff
@@ -171,6 +171,134 @@ class CastExprTest : public functions::test::CastBaseTest {
     assertEqualVectors(expected, result);
   }
 
+  template <typename T>
+  void testDecimalToFloatCasts(TypePtr type) {
+    // short to short, scale up.
+    auto shortFlat = makeNullableShortDecimalFlatVector(
```

Comment:
@ebeworld Can you use the standard vector API?
We are removing the Decimal Vector API in https://github.com/facebookincubator/velox/pull/5571